.editorconfig

```diff
@@ -0,0 +1 @@
+insert_final_newline = true
```

Comment:
:rofl: Irony :rofl:

Reply:
Yeah, my editor doesn't seem to be actually doing it... I'll need to go through the files I changed and update them.